app/views/accounts/actions/_manage_backup_codes.html.slim

```diff
@@ -0,0 +1,4 @@
+= link_to 'backup_code_setup' do
+  span.hide
+    = t('account.items.personal_key')
+  = t('forms.buttons.manage')
```

Comment:
Since this actually generates the backup codes when they don'e exist, lets call this one "Generate". If they do exist, how do we feel about something like "Regenerate" or "Refresh"?